test/semantics/doconcurrent05.f90

```diff
@@ -11,21 +11,17 @@
 ! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 ! See the License for the specific language governing permissions and
 ! limitations under the License.
-
-! RUN: ${F18} -funparse-with-symbols %s 2>&1 | ${FileCheck} %s
-! CHECK: exit from DO CONCURRENT construct \\(mydoc: do concurrent\\(j=1:n\\)\\) to construct with name 'mydoc'
-! CHECK: exit from DO CONCURRENT construct \\(mydoc: do concurrent\\(j=1:n\\)\\)
-! CHECK: exit from DO CONCURRENT construct \\(mydoc: do concurrent\\(j=1:n\\)\\) to construct with name 'mytest3'
-! CHECK: exit from DO CONCURRENT construct \\(do concurrent\\(k=1:n\\)\\)
-! CHECK: exit from DO CONCURRENT construct \\(do concurrent\\(k=1:n\\)\\) to construct with name 'mytest4'
-! CHECK: exit from DO CONCURRENT construct \\(mydoc: do concurrent\\(j=1:n\\)\\) to construct with name 'mytest4'
+!
+! C1167 -- An exit-stmt shall not appear within a DO CONCURRENT construct if 
+! it belongs to that construct or an outer construct.
 
 subroutine do_concurrent_test1(n)
   implicit none
   integer :: n
   integer :: j,k
   mydoc: do concurrent(j=1:n)
   mydo:    do k=1,n
+!ERROR: EXIT must not leave a DO CONCURRENT statement
```

Comment:
You need to change `CMakeLists.txt` so this is executed by `test_errors.sh`.

Reply:
Yes.  That was my intention, but I screwed up.  I'll fix that.